roles/lnls-ans-role-pydm/meta/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+galaxy_info:
+  author: Lucas Russo
```

Comment:
(@fernandohds564 , @lerwys)
@fernandohds564 , I think you could become author (main responsable) for this role, even though it uses a previous @lerwys role as model/template. Is this standard, @lerwys ?

Reply:
Sure. You can put yourself as the author. The author name here is mainly (I think) used by https://galaxy.ansible.com/ (we are not using it), but it serves as a documentation as well.

Reply:
done!